test/Directory.Build.targets

```diff
@@ -0,0 +1,20 @@
+<Project>
```

Comment:
@AArnott this is needed if I run only `RunTests` target with a clean repo, right?

Reply:
By "this" are you referring to this file? 
It's independent of the `RunTests` target in your `build.proj` file (which I'm working to make less and less necessary).
This file teaches your test projects how to build the msbuild tasks first and then consume the output of that build immediately as part of the test project's own build. It's what sets the `CoverletToolsPath` property so that you don't have to use `/p:CoverletToolsPath` manually on the command line.

Reply:
ok thank's.